docs/application/dotnet/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,142 @@
+
+-   Getting component information
+
+    For components that are installed but not necessarily running, you can retrieve information with the [Tizen.Applications.ComponentBased.ComponentInfo](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentInfo.html) class.
+
+Iterator methods are used to travel through a list of components. The `GetRunningComponentsAsync()` method of the `Tizen.Applications.ComponentBased.ComponentManager` class is used for the running components and the `GetInstalledComponentsAsync()` method is used for the installed components.
+
+## Prerequisites
+
+To enable your application to use the component management functionality:
+
+1.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentManager](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ApplicationManager.html), [Tizen.Applications.ComponentBased.ComponentRunningContext](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentRunningContext.html), and [Tizen.Applications.ComponentBased.ComponentInfo](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentInfo.html) classes, include the [Tizen.Applications.ComponentBased](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.html) namespace in your application:
+
+    ```
+    using Tizen.Applications.ComponentBased;
+    ```
+
+2.  To use classes and methods, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+       <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+    </privileges>
+    ```
+
+<a name="manage_context"></a>
+## Managing Component Running Context
+
+To get the component running context and its details, and to operate on the context:
+
+1.  Get the context of the currently running component by creating an instance of the [Tizen.Applications.ComponentBased.ComponentRunningContext](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.ComponentBased.ComponentRunningContext.html) class, with the ID of the component from which the context is being obtained as a parameter.
+
+    To get a component's context, the component must be running.
+
+    ```
+    ComponentRunningContext compoRunningContext = new ComponentRunningContext(Your Component ID);
+    ```
+
+2.  Operate on the context:
+    -   Get the component ID, application ID, and instance ID from the context:
+
+        ```
+        string componentId = compoRunningContext.ComponentId;
+        string applicationId = compoRunningContext.ApplicationId;
+        string instanceId = compoRunningContext.InstanceId;
+        ```
+
+    -   Verify the state of the component:
```

Comment:
@surbhig12, would you please refer https://github.com/Samsung/tizen-docs/pull/840#discussion_r330375642?

To me it looks like the component states must be checked, not verified (as I saied, "verify" sound like "prove" to me).